src/cslStyles.js

```diff
@@ -74,19 +74,21 @@ define(['src/urlUtils', 'src/debug'], function (CSLEDIT_urlUtils, debug) {
 		return styleTitle
 			.replace(/&/g, "and")
 			.replace(/\([A-Z]*\)/g, "") // remove upper case text (acronyms) in parentheses
+			.replace(/\([^\(]*\)$/, "")) //remove content between last set of parentheses
+			.replace(/\[[^\[]*\]$/, "")) //remove content between last set of square parentheses
```

Comment:
Yes, this looks like an error. As I wrote, I haven't had the possibility to test this. Sorry.